backend-schedule/src/main/java/eu/socialedge/hermes/backend/schedule/domain/exception/NotFoundException.java

```diff
@@ -0,0 +1,20 @@
+package eu.socialedge.hermes.backend.schedule.domain.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class NotFoundException extends RuntimeException {
```

Comment:
Imho it's not necessary to create yet another exception. You can just return Resource.notFound():
https://github.com/socialedge/vega/blob/develop/backend-account/src/main/java/eu/socialedge/vega/backend/account/application/api/operator/OperatorController.java#L62